### PR TITLE
break down NewStatusVerifier

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

**Please provide a description of this PR:**
Break `NewStatusVerifier` down as this function does too much things. See [todo](https://github.com/istio/istio/blob/master/istioctl/pkg/verifier/verifier.go#L70)

### CODE REVIEW

File: istioctl/pkg/verifier/verifier.go [hunk 1]
```diff
@@ -65,30 +65,77 @@ type StatusVerifier struct {
...
-		manifestsPath:    manifestsPath,
-		filenames:        filenames,
-		controlPlaneOpts: controlPlaneOpts,
-		logger:           logger,
-		client:           client,
-		iop:              installedIOP,
-		successMarker:    "✔",
-		failureMarker:    "✘",
-	}, nil
+func NewStatusVerifier(verifiers ...Verifier) (*StatusVerifier, error) {
```
[the remark this answers is not in the document]
Comment (kerthcet) [reply istioctl/pkg/verifier/verifier.go]: Or let's just stay the same and remove the TODO. To me, it's acceptable.

**Review by howardjohn — APPROVED** [review]

**Comment by kerthcet** [comment]
ping @istio/wg-user-experience-maintainers

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**